website/docs/tutorials/07-testing/02-cheatcodes/reflect.md

```diff
@@ -0,0 +1,130 @@
+# `reflect`
+```python
+def reflect(self, ids: VmConsts) -> Reflector:
+
+class Reflector:
+    def get(self)
+```
+Loads specified Cairo object into Python type. ```get()``` can return:
+
+- int (for felt)
+- RelocatableValue (for pointer)
+- Cairo Struct (for more complex types)
+
+You can use it to print cairo data and compare complex structures.
+
+```cairo
+%lang starknet
+
+struct SimpleStruct:
+    member x: felt
+end
+
+@external
+func test_reflect_simple():
+    alloc_locals
+
+    local simple_struct: SimpleStruct = SimpleStruct(x=10)
+
+    %{
+        simple_struct = reflect(ids).simple_struct.get()
+        print(simple_struct)
+        # output:
+        # CairoStruct(
+        #     x=10
+        # )
+
+        assert simple_struct.x == 10 # true
```

Comment:
```suggestion
        assert simple_struct.x == 10
```

IMHO, no need for that `true` and if you want to show `false`, you can always `assert not (...)`